### PR TITLE
fix `alias_method_chain' error with rails3+

### PR DESCRIPTION
**Pull request by tainewoo**
fix undefined method `alias_method_chain' error with rails3+, redmine4.x

With redmine4, there would be error, because alias_method_chain is not available after rails3.
NoMethodError: undefined method `alias_method_chain'

http://railscasts.com/episodes/232-routing-walkthrough-part-2?view=asciicast
"This technique is used often in the Rails 3 source code. Earlier versions of Rails used alias_method_chain to override specific behaviour, but now in Rails 3 we can just use super."

Tested work well with Redmine4.x, but i didn't test it with Redmine3.x, so please consider review it or leave it there.

This PR fixes issue https://github.com/HugoHasenbein/redmine_more_filters/issues/4

### CODE REVIEW

File: lib/redmine_more_filters/patches/query_patch.rb [hunk 1]
```diff
@@ -28,9 +28,20 @@ def self.included(base)
         base.class_eval do
           unloadable
           
-          alias_method_chain :sql_for_field, :more_filters
-          alias_method_chain :sql_for_custom_field, :more_filters
-          alias_method_chain :statement, :more_filters
+          #alias_method_chain :sql_for_field, :more_filters
+          alias_method :sql_for_field_without_more_filters, :sql_for_field
+          alias_method :sql_for_field, :sql_for_field_with_more_filters
+          alias_method :more_filters, :sql_for_field
```
Comment (HugoHasenbein) [inline lib/redmine_more_filters/patches/query_patch.rb:34]: this should not be there

File: lib/redmine_more_filters/patches/query_patch.rb [hunk 2]
```diff
@@ -28,9 +28,20 @@ def self.included(base)
...
+          
+          #alias_method_chain :sql_for_custom_field, :more_filters
+          alias_method :sql_for_custom_field_without_more_filters, :sql_for_custom_field
+          alias_method :sql_for_custom_field, :sql_for_custom_field_with_more_filters
+          alias_method :more_filters, :sql_for_custom_field
```
Comment (HugoHasenbein) [inline lib/redmine_more_filters/patches/query_patch.rb:39]: this should not be there

File: lib/redmine_more_filters/patches/query_patch.rb [hunk 3]
```diff
@@ -28,9 +28,20 @@ def self.included(base)
...
+          
+          #alias_method_chain :statement, :more_filters
+          alias_method :statement_without_more_filters, :statement
+          alias_method :statement, :statement_with_more_filters
+          alias_method :more_filters, :statement
```
Comment (HugoHasenbein) [inline lib/redmine_more_filters/patches/query_patch.rb:44]: this should not be there

**Comment by tainewoo** [comment]
cool~ i can close my PR and issue now.
But seems u reverted the locale file of zh.yml to the old version

**Comment by HugoHasenbein** [comment]
can you send the locale zh.yml again? I have accidentally overridden the file

**Comment by tainewoo** [comment]
sure i will, but i’m now using my phone, i will send it again hours later when i can access my computer

**Comment by tainewoo** [comment]
Hi, i created a new PR for that, please review

**Pull request closed without merging**